packages/modules/src/modules/banners/electionAuMoment/components/getElectionAuMomentBanner.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { neutral, space, breakpoints } from '@guardian/src-foundations';
+import { from } from '@guardian/src-foundations/mq';
+// import { Container } from '@guardian/src-layout';
```

Comment:
I think it might be worth using the `Container` component here - we usually center the banner at all the different breakpoints so it's pretty convenient for that

Reply:
I actually don't like the effect of it center-ing itself between breakpoints, so I took this out and replicated the padding from the source component, and then applied `margin: 0 auto;` from `wide`. It's okay at the specific breakpoints, but in between there's a strange left margin that looks awkward. Happy to show you what I mean if this doesn't make sense.

Reply:
Actually, I realised you're completely right! I was looking at how weird it looked in Storybook, but I realised that's out of context of the page. When I played around with it on code it looked totally off. I have reinstated the `Container`.